app/Http/Controllers/Api/UserController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Gate;
+
+class UserController extends Controller
+{
+    /**
+     * @var Response
+     */
+    protected $unauthorized;
+
+    /**
+     * Class constructor
+     */
+    public function __construct()
+    {
+        $this->unauthorized = response()->json(['status' => 'unauthorized'], 403);
+
+        // Only accept logged in Users.
+        $this->middleware(function ($request, $next) {
+            if ($request->user() === null) {
+                return $this->unauthorized;
+            }
+            return $next($request);
+        });
```

Comment:
@tristan-orourke So I think I skipped that one because we're using a custom `redirectTo()` in our auth middleware that just redirects the user to a login page if they're not authenticated. Unless it doesn't fire for API requests?

Reply:
That's a good point. I'm not sure how it works for ajax requests.

Reply:
This is the kind of thing I was concerned about having API routes/controllers mixed in with the rest of the web routes... It's going to get more complicated and convoluted mixing up request and response types and attempting to use the same middleware everywhere.

Laravel has conventions in place for handling web and api requests separately, and I think we should be refactoring towards that in the future.

Reply:
Good point. If we were to set up a different auth guard for the api routes, the 'auth' middleware might work more appropriately for them.